src/index.html

```diff
@@ -1,13 +1,356 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="shortcut icon"
+      href="/images/favicon.ico"
+      type="image/x-ico"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="/" class="logo">
+              <img
+                src="images/logo.svg"
+                alt="BOSE"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="top-actions__menu">
+              <a href="tel:+1 234 5555-55-55" class="icon icon--phone header__phone"></a>
+              <a href="#menu" class="icon icon--menu header__menu-opener"></a>
+            </div>
+
+          </div>
+
+          <h1 class="header__title">
+            The world shades.
+            <br>
+            Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class=" menu__top top-actions">
+            <a href="/" class="logo">
+              <img
+                src="images/logo.svg"
+                alt="BOSE"
+                class="logo__image"
+              >
+            </a>
+
+            <div class="top-actions__menu">
+              <a href="#" class="icon icon--close"></a>
+            </div>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#home" class="menu__link">Home</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About as</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#on-view" class="menu__link">Now on view</a>
+            </li>
+
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">Testimonials</a>
```

Comment:
It doesn't work, maybe you use incorrect href

Reply:
there is no testimonials section in the layout, so I didn't add a link for it(